config.mk

```diff
@@ -30,6 +30,12 @@ VALGRIND ?= valgrind
 # You can also use "make WAYLAND=0" to build without wayland
 # WAYLAND ?= 0
 
+# Disable dependency on libnotify. this will remove the ability
+# to easily send notifications using dunstify or notify-send.
+# Do this if you have your own utility to send notifications.
+# Other applications are fine.
+# LIBNOTIFY ?=0
```

Comment:
Forgot to change the naem here

Reply:
oih n yg ydogaywe7guwryg6ygbjkmsdhagby r7eyhgubdfakahsbyfdbh jkdfgygbyhweraugjk6fdyh jzxcvlg6yrweigjhketubujfdzkl;g6xbycvoikbuyysgbivjshyikxvcyhjm kweriu6ygbklzxc6byyseruigoy6b cvjmk.kgb6yvhrsljkb6 vyhgkl6yberfhn jgbrweujmgfb
cb